### ISSUE TITLE
Accessibility improvements on Lesson Plan landing page

### ISSUE THREAD

**Issue by adamwoodnz** [WordPress/Learn#944]
https://learn.wordpress.org/lesson-plans/ could benefit from the following a11y improvements:

1. The dashicons associated with topics/audiences, etc., should use aria-hidden="true" to prevent the possibility they could be read as characters.
2. Focus states are kind of minimal; the single-pixel dotted outline offers poor visibility. (They also end up looking accidental, rather than designed.)
3. The duration filters would benefit from human-readable labels, e.g. '15 minutes'.  Also, "any" and "over 60m" are not the same thing, so that label feels ambiguous.
4. "Search lesson plans" has a label of "Search for". This results in reading that's "Search landmark. Search for: Search lesson plans". It's very verbose. Would be better if there was just a visible label with "Search lesson plans" and no placeholder.
5. Hit area for links is small and should be enlarged to cover entire tile:

<img width="400" alt="Screen Shot 2022-09-16 at 7 19 56 AM" src="https://user-images.githubusercontent.com/1657336/190518814-6d655f22-345b-400b-9765-55ca866ce895.png">

See https://wordpress.slack.com/archives/C02RP4X03/p1663279177628089

**Comment by StevenDufresne** [comment]
The target zones for clicking are somewhat unpredictable, is it possible to make the whole tile clickable?

<img width="400" alt="Screen Shot 2022-09-16 at 7 19 56 AM" src="https://user-images.githubusercontent.com/1657336/190518814-6d655f22-345b-400b-9765-55ca866ce895.png">

**Comment by adamwoodnz** [comment]
> The target zones for clicking are somewhat unpredictable, is it possible to make the whole tile clickable?

Yes I noticed that too and was going to create another ticket for some UI tweaks but it probably makes sense to bundle that in here too 🙏
